### PR TITLE
FullWidth and Padding params added

### PR DESCRIPTION
**Pull request by JEuler**
New params for Toast customization.

### CODE REVIEW

**Comment by JEuler** [comment]
Can you merge? :)

**Comment by JEuler** [comment]
Any news? :)

**Comment by CaiJingLong** [comment]
Sorry for taking so long to see your PR.

I think there are several points that need to be modified.
I will give you suggestions in the code comments.

File: lib/src/core/toast.dart [hunk 1]
```diff
@@ -40,6 +39,8 @@ ToastFuture showToast(
   OKToastAnimationBuilder animationBuilder,
   Duration animationDuration,
   Curve animationCurve,
+  double toastPadding,
```
Comment (CaiJingLong) [inline lib/src/core/toast.dart:42]: I think toastPadding is a property that constrains the position of toast, so I think we should use EdgeInsetsGeometry.

**Comment by CaiJingLong** [comment]
An additional suggestion, if you are sure to add these two properties, I think they are also added to the OKToast constructor and add a default behavior.

**Comment by JEuler** [comment]
Hm, okay, I will rework solution in free time! Thx.

**Comment by JEuler** [comment]
Hi! I have updated code. :)

**Comment by JEuler** [comment]
@CaiJingLong Could you have a look? :)

**Review by dooooofhgf — APPROVED** [review]

**Comment by AlexV525** [comment]
@JEuler Can you merge from the main branch so we can take another look?

**Comment by JEuler** [comment]
Hi! I will do it later this week, I'm on vacation. :)

**Comment by JEuler** [comment]
Whoops, wrong rebase thing happened, will reopen this :D

**Comment by JEuler** [comment]
https://github.com/OpenFlutter/flutter_oktoast/pull/83

**Pull request closed without merging**